packages/notebook/src/searchprovider.ts

```diff
@@ -236,6 +236,9 @@ export class NotebookSearchProvider extends SearchProvider<NotebookPanel> {
       output: {
         title: trans.__('Search Cell Outputs'),
         description: trans.__('Search in the cell outputs.'),
+        disabledDescription: trans.__(
+          'Search in the cell outputs. (Not available in find and replace mode.)'
```

Comment:
What do you think about:

```suggestion
          'Search in the cell outputs (not available in replace mode).'
```

I was also thinking about being more suggestive of action user needs to take, like "not available when the replace bar is expanded".